neevaco/neeva-android
# Fix obfuscated user id cancellation bug

Because `neevaUser.fetch()` can get called twice and that triggers `fetchObfuscatedUserID()` to get called twice in succession, there is will be two InitializeGooglePlaySubscription Apollo calls. The way Apollo works with these two calls is that the first one will return a `CancellationException`. This PR handles this case + fixes a WelcomeFlow bug that incorrectly shows that the ObfuscatedUserID is null 

## Checklists

### Pointers to help with review
* https://www.figma.com/file/dylvDPB69JG9U1rzZ3HQ52/Android-Premium?node-id=839%3A5094&t=6KMtM6EsIEdI54JP-0

### How was this tested?

### Manual test cases
See video below running through FirstRun and expecting a BillingFlow after the login loading screen.

### Screenshots and screen recordings

https://user-images.githubusercontent.com/20916043/221329835-266efc8f-d55e-43a9-b9b1-b96c71b15868.mp4
